client/core/trade.go

```diff
@@ -533,9 +532,8 @@ func (t *trackedTrade) isActive() bool {
 	t.mtx.RLock()
 	defer t.mtx.RUnlock()
 
-	// Status of the order itself.
-	if t.metaData.Status == order.OrderStatusBooked ||
-		t.metaData.Status == order.OrderStatusEpoch {
+	// Status of the order itself. Epoch and Booked status orders are active.
+	if t.metaData.Status <= order.OrderStatusBooked {
```

Comment:
This kind of integer math is fragile since OrderStatus is to be treated like an enum.  When reasonable, I like to avoid it.  It's also more clear about the intent to list both statuses explicitly.

Reply:
Hmm. I see your reasoning now. I wondered why you separated them, I now think other similar usages should do the same separation. Will update.